src/views/DetailView.js

```diff
@@ -43,4 +43,20 @@ export default class DetailView extends VivView {
 
     return layers;
   }
+
+  /**
+   * Create a viewState for this class, checking the id to make sure this class and viewState match.
+   * @param {Object} args
+   * @param {ViewState} args.ViewState ViewState object.
+   * @returns {ViewState} ViewState for this class (or null by default if the ids do not match).
+   */
+  filterViewState({ viewState, currentViewState }) {
+    if (viewState.id === 'overview') {
+      const target = viewState.navigation?.target;
```

Comment:
```suggestion
      const { target } = viewState;
```
Going along with the other requested change, but I may have this wrong again.  Also could we put the `id`s in a `constants.js` file now that they are being cross-referenced?  I think just `DetailView`'s id (which is `detail`) and `OverviewView`'s (which is `overview`) should be there.  Thanks!